sstables/kl/writer.cc

```diff
@@ -662,7 +662,7 @@ void sstable_writer_k_l::consume_end_of_stream()
         features.disable(sstable_feature::NonCompoundRangeTombstones);
     }
     run_identifier identifier{_run_identifier};
-    _sst.write_scylla_metadata(_pc, _shard, std::move(features), std::move(identifier));
+    _sst.write_scylla_metadata(_pc, _shard, std::move(features), std::move(identifier), {});
```

Comment:
we could have prepended the type std::optional<scylla_metadata::large_data_stats> to {}, to make it easier for the reader, but this shouldn't block this series

Reply:
yeah, but it's a bit of an overkill and reduce readability if done inline IMO.
Maybe on the previous statement we could prepare a disengaged optional ld_stats and move it here.